src/DefaultHandle.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+
+import { DEFAULT_HANDLE_WIDTH_UNITS, BACKGROUND_HEIGHT_UNITS } from './constants/SliderConstants';
+import handlePropTypes, { handleDefaultProps } from './propTypes/HandlePropTypes';
+
+export const propTypes = {
+  ...withStylesPropTypes,
+  ...handlePropTypes,
+  'aria-valuetext': PropTypes.string,
+  'aria-label': PropTypes.string,
+  getLabel: PropTypes.func,
+};
+
+const defaultProps = {
+  ...handleDefaultProps,
+  'aria-valuetext': undefined,
+  'aria-label': undefined,
+  getLabel: undefined,
+};
+
+function DefaultHandle({
+  css, styles, orientation, disabled, ...passProps
+}) {
+  const {
+    handleRef, theme, getLabel, ...rest
+  } = passProps; // eslint-disable-line no-unused-vars
+  return (
+    <button
+      ref={handleRef}
+      {...css(
+        styles.DefaultHandle_handle,
+        orientation === 'vertical' ? styles.DefaultHandle_handle__vertical : styles.DefaultHandle_handle__horizontal,
+        disabled && styles.DefaultHandle_handle__disabled,
+      )}
+      {...rest}
+    />
+  );
+}
+DefaultHandle.propTypes = {
```

Comment:
`DefaultHandle.propTypes = propTypes`? I don't think there's a reason to spread this on